packages/transactions/tests/payload.test.ts

```diff
@@ -117,6 +118,30 @@ test('Smart contract payload serialization and deserialization', () => {
   expect(deserialized.codeBody.content).toBe(codeBody);
 });
 
+test('Versioned smart contract payload serialization and deserialization', () => {
+  const contractName = 'contract_name';
+  const codeBody =
+    '(define-map store ((key (buff 32))) ((value (buff 32))))' +
```

Comment:
As far as I can tell this is the came contract as `fs.readFileSync('./tests/contracts/kv-store.clar')`, we could reuse that file for this test as well...